docs/source/api.rst

```diff
@@ -13,25 +13,26 @@ Prediction
 .. autosummary:: compute_persistence
 .. autosummary:: compute_uninitialized
 
+
 Bootstrap
 ~~~~~~~~~
 .. currentmodule:: climpred.bootstrap
 
-.. autosummary:: bootstrap_uninitialized_ensemble
-.. autosummary:: bootstrap_uninit_pm_ensemble_from_control
 .. autosummary:: DPP_threshold
 .. autosummary:: varweighted_mean_period_threshold
-.. autosummary:: bootstrap_compute
+.. autosummary:: bootstrap_perfect_model
```

Comment:
Make sure to add these as `.. autofunction` at the bottom. I think they should also be documented exclusively (or through docstring inheritance if you figure it out) because currently the API just tells users that it's a wrapper